.github/workflows/python-package.yaml

```diff
@@ -7,6 +7,7 @@ on:
 
 env:
   IMAGE_TAG: protoc-gen-validate:${{ github.sha }}
+  REF_NAME: ${{ github.ref_name }}
```

Comment:
Can we just use `${{ github.ref_name }}` below? I'm not sure what the indirection buys us.